utils/src/tree/AzExtTreeItem.ts

```diff
@@ -52,7 +52,7 @@ export abstract class AzExtTreeItem implements types.AzExtTreeItem {
             return undefined;
         }
 
-        if (this.treeDataProvider.collapsibleStateTracker) {
+        if (this.parent && this.treeDataProvider.collapsibleStateTracker) {
```

Comment:
Temporary work-in that I had for testing while we discuss... not meant to be the final state.